src/chi/components/tabs/tabs.scss

```diff
@@ -55,15 +63,22 @@
           margin-left: 1.5rem;
         }
 
-        &.-active > a {
-          border-bottom: 0.25rem solid $brand-color;
-          padding-bottom: 1rem;
-        }
-
         & > a {
           line-height: $line-height-smaller;
           padding: 1.25rem 0;
         }
+
+        &.a-tabs__sliding-border {
+          margin-left: 0;
+          height: 0.25rem;
```

Comment:
Can we change the sliding borders height value to 0.125rem (2px) and apply that not just here but on the non-js active tab as well?